### DISCUSSION TITLE
Is there a way to configure the plugin in a windows / wsl environment ?

### DISCUSSION THREAD

**Discussion by scaturigo** [ocamllabs/vscode-ocaml-platform#810] [Q&A] [answered]
Hi,

Thanks for this nice plugin.
Is it possible to use the plugin on windows with an ocaml wsl installation ?

I tried with 
{
    "ocaml.useOcamlEnv": false,
    "ocaml.repl.path": "bash -ic ocaml"
}
But it's not working...

**Comment by mlantas** [comment, marked as answer]
For WSL, you should use the [Remote WSL extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl).

It will let you open a folder with WSL and use `ocaml` directly instead of calling it through `bash.exe`.

**Reply by scaturigo** [reply]
That is a good solution. thank you!
